recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ requirements:
     - zarr
     - fsspec
     - s3fs
-    - xarray-datatree ==0.0.4
+    - xarray-datatree ==0.0.6
```

Comment:
```suggestion
    - xarray-datatree ==0.0.6
    - cmocean
```

Reply:
Awesome, thanks! I was just going to ping you :D